app/forms/claimants_detail.rb

```diff
@@ -0,0 +1,12 @@
+class ClaimantsDetail < BaseForm
+  attribute :claimants_name, :string
+  attribute :early_conciliation_details, :boolean
```

Comment:
Maybe rename these booleans - they do not suggest the 'yes / no' type of field - for example 'early_conciliation_details' suggests the details - but this flag is not really the details